### PR TITLE
TKTS-001: Add boilerplate to test.html

### PR DESCRIPTION
**Pull request by just-hugo**
[TKTS-001](url) is part of a [larger epic](url) to add content to test.html. This ticket prepares the file with boilerplate. The boilerplate came from our internal [style guide](url).

### CODE REVIEW

File: html/test.html [hunk 1]
```diff
@@ -1 +1,11 @@
-
+<!DOCTYPE html>
```
Comment (just-hugo) [inline html/test.html:1]: This is the boilerplate I added.

File: html/test.html [hunk 2]
```diff
@@ -1 +1,11 @@
...
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>HTML 5 Boilerplate</title>
```
Comment (thagomized) [inline html/test.html]: Update the title to the name of the feature that's going to be built on test.html.

**Review by thagomized — CHANGES_REQUESTED** [review]

File: html/test.html (hunk 2 above)
Comment (just-hugo) [reply html/test.html]: Good catch, thanks. Updated.

**Review by thagomized — APPROVED** [review]
Looks good to me.

**Pull request merged**